include/fastdds/dds/publisher/DataWriter.hpp

```diff
@@ -113,73 +109,86 @@ class DataWriter : public DomainEntity
      * Write data with handle.
      * @param data Pointer to the data
      * @param handle InstanceHandle_t.
-     * @return True if correct
-     * @par Calling example:
-     * @snippet fastrtps_example.cpp ex_PublisherWrite
+     * @return RETCODE_PRECONDITION_NOT_MET if the handle introduced does not match with the one associated to the data,
+     * RETCODE_OK if the data is correctly sent and RETCODE_ERROR otherwise.
```

Comment:
Maybe it is not yet in the implementation, but this method could return TIMEOUT on some cases

Reply:
That ReturnCode is not returned by the moment